auto-notify.plugin.zsh

```diff
@@ -40,7 +40,7 @@ function _auto_notify_message() {
         if [[ "$exit_code" != "0" ]]; then
             urgency="critical"
         fi
-        notify-send "$title" "$body" "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
+        notify-send "$title" "$body" --app-name=zsh "--urgency=$urgency" "--expire-time=$AUTO_NOTIFY_EXPIRE_TIME"
```

Comment:
nicely done :) might be even better to allow this to be configurable but I'll put that into a next step